integration_tests/booktype_gorm_composite/test_starter.go

```diff
@@ -0,0 +1,63 @@
+package booktype_gorm_composite
+
+import (
+	"context"
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/merlinapp/datarepo-go"
+	"github.com/merlinapp/datarepo-go/cachestore/composite"
+	"github.com/merlinapp/datarepo-go/cachestore/memory"
+	"github.com/merlinapp/datarepo-go/cachestore/nocache"
+	"github.com/merlinapp/datarepo-go/cachestore/stats"
+	"github.com/merlinapp/datarepo-go/integration_tests"
+	"github.com/merlinapp/datarepo-go/integration_tests/booktype_gorm_composite/testdomain"
+	"github.com/merlinapp/datarepo-go/integration_tests/model"
+	gorm2 "github.com/merlinapp/datarepo-go/repo/gorm"
+	"time"
+)
+
+var testInstance *testdomain.SystemInstance
+
+func startSystemForIntegrationTests() *testdomain.SystemInstance {
+	if testInstance != nil {
+		return testInstance
+	}
+
+	db := integration_tests.TestConnectionFactory()
+	db.LogMode(true)
+
+	memoryCacheStore := memory.NewFreeCacheInMemoryStore(1 * 1024 * 1024)
```

Comment:
maybe convert those integer values to variables to know what it is
```suggestion
	memoryCacheStore := memory.NewFreeCacheInMemoryStore(1 * 1024 * 1024)
```